robocop/rules.py

```diff
@@ -137,3 +137,14 @@ def __lt__(self, other):
 
     def get_fullname(self):
         return f"{self.severity.value}{self.rule_id} ({self.name})"
+
+    def to_json(self):
+        return {
+                "source": self.source,
+                "line": self.line,
+                "col": self.col,
+                "severity": self.severity.value,
+                "rule_id": self.rule_id,
+                "desc": self.desc,
+                "name": self.name
```

Comment:
I'm still not 100% sure about the naming here but it looks ok so we can go with it.